src/config/index.ts

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
```

Comment:
Don't import the whole lodash package, it's not the good practice
```suggestion
import isEmpty from 'lodash/isEmpty';
```